_posts/2021-04-21-the-definitive-guide.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "Trino: The Definitive Guide"
+author: Matt Fuller, Manfred Moser and Martin Traverso
+excerpt_separator: <!--more-->
+---
+
+Just over a year ago we [announced the availability of the first book about
+Trino]({{ site.url }}{% post_url 2020-04-11-the-definitive-guide %}) - our
+definitive guide. Back then the project was still called Presto, and the rename
+with the end of 2020 was a good reason for us to give the book a refresh.
+
+Today, we are happy to announce that a new edition now titled __Trino: The
+Definitive Guide__ is available.
+
+> ## [Get a free copy of Trino: The Definitive Guide](https://www.starburst.io/info/oreilly-trino-guide/) from [Starburst](https://www.starburst.io) now!
+
+<!--more-->
+
+<img src="/assets/ttdg-cover.png" align="right" style="float: right; margin-left: 20px; margin-bottom: 20px; width: 100%; max-width: 350px;">
+
+The new edition of the book from O'Reilly is again available in digital formats
```

Comment:
*again* doesn't seem necessary in the sentence.

Reply:
fair